schema_based_on_diagram.sql

```diff
@@ -0,0 +1,44 @@
+-- Table: Patient
+CREATE TABLE Patient (
+  id INT PRIMARY KEY,
+  name VARCHAR(100),
+  date_of_birth DATE
+);
+
+-- Table: Medical_Histories
+CREATE TABLE Medical_Histories (
+  id INT PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id INT,
+  status VARCHAR(100),
+  FOREIGN KEY (patient_id) REFERENCES Patient(id)
+);
+
+-- Table: Invoices
+CREATE TABLE Invoices (
+  id INT PRIMARY KEY,
+  total_amount DECIMAL(10, 2),
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INT,
+  FOREIGN KEY (medical_history_id) REFERENCES Medical_Histories(id)
+);
+
+-- Table: Treatments
+CREATE TABLE Treatments (
+  id INT PRIMARY KEY,
+  type VARCHAR(100),
+  name VARCHAR(100)
+);
+
+-- Table: Invoice_Items
```

Comment:
Hello, thank you for submitting your project. You have done a good job of creating the tables and setting the primary and foreign keys. However, you have missed one requirement: creating indexes for all the foreign keys. Indexes are important because they help speed up the queries that involve joining tables by foreign keys. Without indexes, the database would have to scan the whole table to find the matching rows, which can be slow and inefficient. To create an index, you can use the CREATE INDEX ON table_name(column_name) syntax. For example, to create an index on the patient_id column in the medical_histories table, you can write:

CREATE INDEX ON medical_histories (patient_id);

Please add the indexes for all the foreign keys in your code and resubmit your project for another review. This will improve the performance of your database schema. If you have any questions, feel free to ask. Good luck! 😊

Reply:
Hi @topeogunleye,

Thank you for your feedback. Similar to the previous issues, this happened because of my failure to push the local changes to the PR. I have now updated the PR, and the indexes for foreign keys have been added.